setup/express.js

```diff
@@ -0,0 +1,122 @@
+const expressWinston = require('express-winston');
+
+const { setupNodeLogger } = require('./winston');
+const { DEFAULT_LOG_LEVEL } = require('..');
+
+const defaultFilters = [
+  { agent: 'Consul Health Check', urls: ['/'], statuses: [200] },
+  { agent: 'Pingdom.com_bot_version', urls: ['/', '/api/v1/health'], statuses: [200] },
+];
+
+function skipRequests(filters) {
+  return (req, res) => {
+    const userAgent = req.headers['user-agent'] || '';
+
+    return filters.some(
+      ({ agent, urls, statuses }) => userAgent.includes(agent)
+        && urls.includes(req.url)
```

Comment:
Um - includes? Find me _one_ URL that doesn't include `/`... perhaps `url.path.endsWith` or its mental equivalent?

Reply:
not sure I understand

Reply:
oh, ok, I think I get it now. you mean that `includes` can be too wide if the input is something like `/`, which it is

Reply:
Ugh, No, my bad, I misinterpreted `includes` to mean "has substring", and not as "array contains". Please disregard.

Reply:
In retrospect my confusion is fairly reasonable; `userAgent.includes(...)` is exactly what I thought it was, a substring test, whereas `urls.includes(...)` actually checks whether the array contains the specified URL precisely. Nothing much to do here, except hate JavaScript with a passion born of a thousand suns.

Reply:
oh, right, and I did it as well. lol